ffi/src/clipboard/mod.rs

```diff
@@ -1,63 +1,15 @@
 pub mod message;
 
+
+#[cfg(target_os = "windows")]
+pub mod windows;
```

Comment:
issue: It’s problematic to conditionally compile Rust code while still generating the bindings for functions that may or may not exist. Let’s just check dynamically and return an error dynamically when the system is not supported. Do not try to add non-diplomat cfgs in Diplomat bindings.

Reply:
updated